docs/api/browser-window.md

```diff
@@ -1648,3 +1648,24 @@ removed in future Electron releases.
 [vibrancy-docs]: https://developer.apple.com/documentation/appkit/nsvisualeffectview?preferredLanguage=objc
 [window-levels]: https://developer.apple.com/documentation/appkit/nswindow/level
 [chrome-content-scripts]: https://developer.chrome.com/extensions/content_scripts#execution-environment
+
+### Properties
+
+#### `win.excludedFromWindowsMenu` _macOS_
```

Comment:
> `excludedFromWindowsMenu`

The naming off this could be confusing considering "Windows" as the OS as well.  No suggestions right now, but I think we need to come up with an alternate here 🤔

Reply:
suggestions:

* `excludedFromListOfWindows`
* `excludedFromListOfWindowsMenu`
* `excludedFromShownWindows`
* `excludedFromShownWindowsMenu`
* `excludedFromShownWindowsList`
* `excludedFromWindowsList`

Reply:
updated with `excludedFromShownWindowsMenu` ☝️